content/news/032/index.md

```diff
@@ -424,6 +424,31 @@ series.
 [matthew-bryant-youtube]: https://www.youtube.com/channel/UC7v3YEDa603x_84PgCPytzA
 [matthew-bryant-github]: https://github.com/mwbryant
 
+### [Bevy Cheatbook]
+
+[Bevy Cheatbook] (by Ida Iyes) had many improvements.
+
+There is a new [guided tutorial page][bevy-cb-tutorial], to help you navigate
+the book in an order that makes sense for learning, starting from beginner
+topics and progressing towards more advanced! There is also a new "getting
+started" page, as an alternative to Bevy's official instructions.
+
+New topics added to the book: working with Bevy Time/Timers/Stopwatches, ECS
+data storage kinds, exclusive systems, direct World access, non-Send types,
+and more…
```

Comment:
I feel like the special character for 3 dots might not be great, not sure though

```suggestion
and more...
```

Reply:
Do you have any reasoning for that?

Usually, I prefer using Unicode punctuation characters rather than "emulating" them with ASCII.

Reply:
I think it's just for consistency (other people generally don't bother going to the effort of doing the fancy punctuation).

That being said, I just spotted that we can make our static site generator automatically do the ASCII-to-fancy-unicode translation if we want ([`smart_punctuation`](https://www.getzola.org/documentation/getting-started/configuration/) setting), so we might want to look at turning that on at some point - then we'd have consistency no matter how people write their sections.

Reply:
Ok, I will leave it up to you folks to decide what to do with the punctuation. Feel free to change it if you want to.

Reply:
I'm impartial, I don't particularly know if one is better, but I was just thinking along the lines of "ASCII where possible" is likely to not have issues :)

I'm fine to leave this, and talk about it in a separate issue :+1: